packages/table/src/constants.ts

```diff
@@ -0,0 +1,12 @@
+const LGID_ROOT = 'lg-table';
+
+export const LGIDS = {
+  root: LGID_ROOT,
+  cell: `${LGID_ROOT}-cell`,
+  row: `${LGID_ROOT}-row`,
+  header: `${LGID_ROOT}-header`,
+  selectAllCheckbox: `${LGID_ROOT}-select_all`,
+  checkbox: `${LGID_ROOT}-checkbox`,
+  sortIcon: `${LGID_ROOT}-sort_button`,
+  expandButton: `${LGID_ROOT}-expand_button`,
+} as const;
```

Comment:
`as const` doesn't have any effect here when you construct the values using `LGID_ROOT`.
These are all still just typed as `string`, not the literal value

Reply:
Am I misinterpreting this? The literal values show up when using `as const`
<img width="557" alt="Screenshot 2024-08-13 at 12 35 07 PM" src="https://github.com/user-attachments/assets/35f5816d-4fd7-4713-9436-b37031b3d3ba">
<img width="362" alt="Screenshot 2024-08-13 at 12 35 21 PM" src="https://github.com/user-attachments/assets/80dc41b9-0b61-4eba-beab-be57604ba336">